conjureup/controllers/clouds/gui.py

```diff
@@ -53,16 +55,27 @@ def render(self):
         excerpt = app.config.get(
             'description',
             "Where would you like to deploy?")
-        view = CloudView(app,
-                         public_clouds,
-                         custom_clouds,
-                         cb=self.finish)
+
+        self.view = CloudView(app,
+                              public_clouds,
+                              custom_clouds,
+                              cb=self.finish)
+
+        if 'localhost' in cloud_types:
+            # Need to load the localhost provider here early so our LXD methods
+            # are available
+            app.provider = load_schema('localhost')
```

Comment:
This seems fragile.  Why not make the `LocalhostProvider` instance a param to the task, rather than sticking it in the global state?